.travis.yml

```diff
@@ -6,24 +6,25 @@ env: SWIFT_VERSION=5.6.2
 vm:
   size: large
 jobs:
-  - name: coverage
-    install:
-      - dotnet tool install -g dotnet-coverage
-      - curl -Os https://uploader.codecov.io/latest/linux/codecov
-      - chmod +x codecov
-    script: PATH+=:$HOME/.dotnet/tools make codecov
-  - name: c, cl, error
-    script: make -j5 test-c test-cl test-error
-  - name: cpp, java
-    install: sudo apt install clang-12
-    script: make -j5 test-cpp test-java CXX='clang++-12 -std=c++20'
-  - name: cs
-    script: make -j5 test-cs
-  - name: js, ts
-    install: nvm install node
-    script: make -j5 test-js test-ts
-  - name: py, swift
-    install:
-      - wget https://swift.org/builds/swift-$SWIFT_VERSION-release/ubuntu2004/swift-$SWIFT_VERSION-RELEASE/swift-$SWIFT_VERSION-RELEASE-ubuntu20.04.tar.gz
-      - tar xf swift-$SWIFT_VERSION-RELEASE-ubuntu20.04.tar.gz -C ..
-    script: make -j5 test-py test-swift SWIFTC=../swift-$SWIFT_VERSION-RELEASE-ubuntu20.04/usr/bin/swiftc
+  include:
```

Comment:
Why add this `include:` ?

Reply:
That's something VSCode suggested. Follwing .travis yml shema. Found this documentation https://docs.travis-ci.com/user/build-stages/#how-to-define-build-stages